packages/login/src/main.js

```diff
@@ -2,17 +2,22 @@ import React from 'react'
 import {appFactory, consoleLogger, externalEvents, errorLogging} from 'tocco-util'
 
 import * as passwordUpdate from './modules/passwordUpdate/dialog/actions'
+import * as pwRequest from './modules/passwordRequest/actions'
+import * as loginUpdate from './modules/login/actions'
```

Comment:
why the "Update" part? You import all actions i dont see why you should add Update.